messaging-experiments-82.yaml

```diff
@@ -233,6 +233,75 @@
   targeting: browserSettings.update.channel == 'release' &&
     localeLanguageCode == 'en' && region == 'US' &&
     'app.shield.optoutstudies.enabled'|preferenceValue
+
+- id: WIZARD-LESS-PASSWORD-IMPORT-AUTOCOMPLETE-EXPERIMENT
+  enabled: true
+  targeting: firefoxVersion >= '82.' &&
+    browserSettings.update.channel != 'release' &&
+    localeLanguageCode == 'en' &&
+    isFirstStartup
```

Comment:
should isFirstStartup targeting change to below to keep enrolled user still in experiment, isFirstStartup will be true for new user only the first time, on subsequent restart of firefox instance isFirstStartUp will evaluate false failing targeting

.. && (isFirstStartup || 'bug-1668315-message-wizard-less-password-import-autocomplete-exper-release-82-83' in activeExperiments)

Reply:
Oh indeed! Fixing